src/components/conversation-card/ConversationCard.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="conversation-card" @click="onClick">
+
+    <div class="avatar-container">
+      <avatar :image="image" />
+    </div>
+
+    <template v-if="!loading">
+      <div class="first">
+        <p class="txt-4 bold">
+          {{ userName }}
+        </p>
+        <p
+          v-if="isSmallOrDown"
+          class="last-message txt-5"
+          >
+          {{ lastMessage }}
+        </p>
+        <p
+          class="light"
+          :class="isSmallOrDown ? 'txt-6' : 'txt-5'"
+          >
+          <template v-if="isSmallOrDown" >
+            <span class="premium">Inquiry Sent</span>
+            <span class="bull">&bull;</span>
+          </template>
+          {{ time }}
+        </p>
+      </div>
+
+      <div
+        v-if="!isSmallOrDown"
+        class="second"
+        >
+        <p class="last-message txt-4">
+          {{ lastMessage }}
+        </p>
+        <p class="txt-5 light">
+          {{ additionnalInfo }}
+        </p>
+      </div>
+
+      <div
+        v-if="!isSmallOrDown"
+        class="third"
+        >
+        <p class="txt-4">
+          {{ price }}
+        </p>
+        <p
+          class="txt-5 bold"
+          :class="statusClass"
+          >
+          {{ status }}
+        </p>
+      </div>
+    </template>
+
+    <p v-else>Loading...</p>
+
+  </div>
+</template>
+
+<script>
+import Avatar from '../avatar'
+import { responsiveHandler } from '../../mixins/responsiveHandler'
+
+export default {
+  name: 'conversation-card',
+  mixins: [ responsiveHandler ],
+  components: { Avatar },
+  props: {
+    image: { type: String, default: '' },
+    userName: { type: String, default: '' },
+    time: { type: String, default: '' },
+    lastMessage: { type: String, default: '' },
+    additionnalInfo: { type: String, default: '' },
+    price: { type: String, default: '' },
+    status: { type: String, default: 'Pending' },
+    statusClass: {
+      type: String,
+      default: 'black',
+      validator: value => ['success', 'warning', 'black'].includes(value)
+    },
+    loading: { type: Boolean, default: false },
+    read: { type: Boolean, default: false }
+  },
+  methods: {
+    onClick() {
+      this.$emit('click')
```

Comment:
@KevinTss can this not result in a conflict by using just `click`, is it not better to use `onClickConversationCard`? Do we also need to emit which card is clicked?

Reply:
No because in vue you can use `@click` and `@click.native` for handle this event.
Is just a best practise that I saw some time ago on an article